pkg/controller/seed-controller-manager/constraint-controller/controller.go

```diff
@@ -180,9 +194,9 @@ func (r *reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 
 func addLabel(constraint *kubermaticv1.Constraint) *kubermaticv1.Constraint {
 	if constraint.Labels != nil {
-		constraint.Labels[Key] = Value
+		constraint.Labels[Key] = constraint.Name
 	} else {
-		constraint.Labels = map[string]string{Key: Value}
+		constraint.Labels = map[string]string{Key: constraint.Name}
```

Comment:
why is this change needed?

Reply:
did this in order to get all the existing default constraints in user-cluster-ns for a request from kubermatic/constraint